### PR TITLE
Test setup 2

### PR DESCRIPTION
**Pull request by gaoqiangzhou**
NO CONFLICTS!!!!!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +9,8 @@
     "build": "vite build",
     "lint": "eslint . --ext js,jsx --report-unused-disable-directives --max-warnings 0",
     "preview": "vite preview",
-    "start": "serve -s dist"
+    "start": "serve -s dist",
+    "test": "jest"
```
Comment (aappacode) [inline package.json:13]: Did you receive any errors in your terminal when you ran this?

**Comment by aappacode** [comment]
@gaoqiangzhou were you able to do any easy/example/unit tests with this setup?

**Comment by gaoqiangzhou** [comment]
> @gaoqiangzhou were you able to do any easy/example/unit tests with this setup?

image
yep

**Review by aappacode — APPROVED** [review]
Unit test example worked with this setup, and no merge conflicts reported!

File: package.json (hunk 1 above)
Comment (aappacode) [reply package.json:13]: Comment was addressed

**Pull request merged**